screens/store/checkout/CheckoutScreen.js

```diff
@@ -4,9 +4,22 @@ import { WebView } from "react-native-webview";
 import { SimpleContainer } from "../../../components/Container/SimpleContainer";
 import { LoadingIndicator } from "../../../components/LoadingIndicator/LoadingIndicator";
 import { styles } from "../../../styles/store/checkout/checkout-screen/styles";
+import Routes from "../../../routes/routes";
 
 export function CheckoutScreen({ navigation, route }) {
-  const { redirectSession } = route.params;
+  const { redirectSession, cameFrom } = route?.params || {};
+  if (!redirectSession) {
+    navigation.navigate(Routes.Cart);
+    return null;
+  }
+
+  const goBack = () => {
+    webviewRef.current.stopLoading();
+    if (cameFrom !== "CartView") navigation.replace("CartView");
```

Comment:
won't this code execute all the navigations because there is no `return` statement? I'm not sure I got why a regular go back won't work but I might be missing something.